From: Sjoerd Simons <sjoerd@collabora.com>
Date: Thu, 9 Apr 2020 09:12:31 +0200
Subject: dependencies: Don't build with flashrom and umock support

For atari there is no need to update devices using flashrom or
thunderbold so drop dependencies.

Signed-off-by: Sjoerd Simons <sjoerd@collabora.com>
---
 contrib/ci/dependencies.xml | 34 ----------------------------------
 1 file changed, 34 deletions(-)

diff --git a/contrib/ci/dependencies.xml b/contrib/ci/dependencies.xml
index 559530d..7b9b4c8 100644
--- a/contrib/ci/dependencies.xml
+++ b/contrib/ci/dependencies.xml
@@ -328,20 +328,6 @@
       <package variant="x86_64" />
     </distro>
   </dependency>
-  <dependency type="build" id="libflashrom-dev">
-    <distro id="fedora">
-      <package>flashrom-devel</package>
-    </distro>
-    <distro id="debian">
-      <control />
-      <package variant="x86_64" />
-      <package variant="i386" />
-    </distro>
-    <distro id="ubuntu">
-      <control />
-      <package variant="x86_64" />
-    </distro>
-  </dependency>
   <dependency type="build" id="gcab">
     <distro id="centos">
       <package />
@@ -1225,26 +1211,6 @@
       <package variant="x86_64" />
     </distro>
   </dependency>
-  <dependency type="build" id="umockdev">
-    <distro id="arch">
-      <package />
-    </distro>
-    <distro id="centos">
-      <package>umockdev-devel</package>
-    </distro>
-    <distro id="fedora">
-      <package>umockdev-devel</package>
-    </distro>
-    <distro id="debian">
-      <control />
-      <package variant="x86_64" />
-      <package variant="i386" />
-    </distro>
-    <distro id="ubuntu">
-      <control />
-      <package variant="x86_64" />
-    </distro>
-  </dependency>
   <dependency type="build" id="valac">
     <distro id="arch">
       <package>vala</package>
